lib/matplotlib/artist.py

```diff
@@ -774,7 +774,7 @@ def set_clip_path(self, path, transform=None):
 
         Parameters
         ----------
-        path : `.Patch` or `.Path` or `.TransformedPath` or None
+        path : `~matplotlib.patches.Patch` or `.Path` or `.TransformedPath` or None
```

Comment:
Yes, this one can be even better, but so far I'm down the search and replace route.